vpn/tunnel.go

```diff
@@ -403,3 +523,17 @@ func quote(key string) string {
 	}
 	return quoted
 }
+
+func hostsToIPStrings(hosts map[dnsname.FQDN][]netip.Addr) []string {
```

Comment:
I'm not familiar with the `dnsname.FQDN` type but I'm assuming this can be either with or without the full domain suffix?

Reply:
We set all the `dnsname.FQDNs` ourselves, using `dnsname.ToFQDN`, and they're all fully qualified. However this function doesn't read any of the fqdn keys, only the ip address values.